### PR TITLE
[TINB-446] Adds Black

### PR DESCRIPTION
**Pull request by diegoponciano**
**What**
Apply black to project files.

**Why**
To maintain the Olist standard.

Task: https://jira-olist.atlassian.net/browse/TINB-446

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -18,6 +19,43 @@ repos:
...
+  hooks:
+    - id: black
+
+- repo: git@github.com:olist/hulks.git
+  rev: 0.4.1
+  hooks:
+    - id: check-filename
+    - id: check-invalid-domains
+    - id: check-logger
+    - id: check-mutable-defaults
```
Comment (thiagolopes) [inline .pre-commit-config.yaml:61]: ta sem \n aq

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -1,9 +1,10 @@
 repos:
--   repo: git@github.com:pre-commit/pre-commit-hooks
-    sha: v1.1.0
-    hooks:
+- repo: git@github.com:pre-commit/pre-commit-hooks
+  rev: v3.1.0
```
Comment (thiagolopes) [inline .pre-commit-config.yaml:3]: esse config é de qual template?

Comment (diegoponciano) [reply .pre-commit-config.yaml:3]: @thiagolopes daqui: https://github.com/olist/package-project-template/blob/master/%7B%7B%20cookiecutter.repo_name%20%7D%7D/.pre-commit-config.yaml

File: .pre-commit-config.yaml (hunk 1 above)
Comment (diegoponciano) [reply .pre-commit-config.yaml]: arrumei

**Pull request merged**